server/src/main/scala/org/http4s/armeria/server/ArmeriaServerBuilder.scala

```diff
@@ -313,6 +313,7 @@ sealed class ArmeriaServerBuilder[F[_]] private (
   def withMeterRegistry(meterRegistry: MeterRegistry): Self =
     atBuild(_.meterRegistry(meterRegistry))
 
+  // Note this effect became uncancelable since Cats-Effect 3.5.0
```

Comment:
Why not just F.fromCompletableFuture here, too?

Reply:
To type fewer 😆  Joking aside, I don't have a strong opinion about why not have `F.fromCompletableFuture` here